### PR TITLE
[Hotfix][Meshing]Compilation error after update ULF Application

### PR DESCRIPTION
**Pull request by loumalouomega**
Minor typo in one declaration. Additionally I cleaned up the old headers

### CODE REVIEW

File: applications/MeshingApplication/external_includes/trigen_droplet_refine.h [hunk 1]
```diff
@@ -109,7 +109,7 @@ class TriGenDropletModeler
 
     //*******************************************************************************************
     //*******************************************************************************************
-    void ReGenerateMeshDROPLET(
+    void ReGenerateMeshDroplet(
```
Comment (loumalouomega) [inline applications/MeshingApplication/external_includes/trigen_droplet_refine.h:112]: @elafnm This is the typo, the rest is just clean up

**Comment by rubenzorrilla** [comment]
I'm approving to avoid the app. to be crashed in the master branch. In any case, @elafnm check that you agree with the changes.

**Review by rubenzorrilla — APPROVED** [review]

File: applications/MeshingApplication/external_includes/trigen_droplet_refine.h (hunk 1 above)
Comment (elafnm) [reply applications/MeshingApplication/external_includes/trigen_droplet_refine.h:112]: Greetings dears

Agreed,

Best regards

**Comment by elafnm** [comment]
thank you dears @loumalouomega  and @rubenzorrilla,

sorry about that typo, that pull request had too many modifications at once. hoping for the next PR to be smoother and more convenient to you. Thank you all for your usual support and appreciated efforts.

also, sorry about the delay in reply but this is due to the time difference,

best regrds,

**Pull request merged**